deploy-board/deploy_board/templates/deploys/deploy_progress.tmpl

```diff
@@ -279,21 +279,23 @@
 </div>
 {% endwith %}
 
-<div class="panel panel-default">
-    <div class="panel-heading clearfix">
-        <h4 class="panel-title pull-left"><i class="fa fa-fw fa-user"></i> Accounts</h4>
-    </div>
-    <div class="panel-body">
-        <div class="row">
-            {% for account in accounts %}
-                <a href="#" class="deployToolTip btn btn-xs btn-default host-btn">
-                    <small>{{ account.name }}</small>
-                    <i class="fa fa-fw fa-user"></i>
-                </a>
-            {% endfor %}
+{% if accounts %}
```

Comment:
add a quick description for the account section such as "AWS account is being used in this stage". Main purpose of the section is to show which account the hosts belong to right?

Reply:
Added fixes:
1. Moved the `Account` panel before the `Hosts` panel.
2. Added tooltip.

### Before mouse hover on the tooltip
![Screenshot 2024-03-14 at 2 06 51 PM](https://github.com/pinterest/teletraan/assets/20383875/49e0d0c1-1edc-40f1-835d-3c731af3c1db)

### After mouse hover on the tooltip
![Screenshot 2024-03-14 at 2 07 15 PM](https://github.com/pinterest/teletraan/assets/20383875/0de45bca-dbda-4ce6-ad53-8b6d218906ba)

Reply:
we might need to make the style of Build section and Account section to be similar, looks like Account is bigger than build while build is the one people care more about.

Reply:
besides, I think it's better to make the account section to not consume lots of area here

Reply:
I changed UI: showed in the meeting